static/redfish/v1/JsonSchemas/OemComputerSystem/index.json

```diff
@@ -116,9 +116,9 @@
             "properties": {},
             "type": "object"
         },
-        "IBMOem": {
+        "IBM": {
```

Comment:
Need to change the XML as well like https://github.com/ibm-openbmc/bmcweb/pull/275/files

Reply:
Thanks, done.